core/authors/views.py

```diff
@@ -184,4 +189,129 @@ def check_is_following(self, request, pk):
         
         return Response({
             "isFollowingUser": follow.exists()
-        }, status=200)
\ No newline at end of file
+        }, status=200)
+
+    # All posts the currently auth'd user can see of pk
+    # /author/{AUTHOR_ID}/posts
+    @action(detail=True, url_path="posts")
+    def posts(self, request, pk=None):
+        if request.user.is_anonymous:
```

Comment:
This should still work if they're anonymous, they would only be able to see public posts though

Reply:
that's depressing